docs/consume-packages/includes/restore-nuget-exe-cli.md

```diff
@@ -1,13 +1,21 @@
-Use the [restore](../../reference/cli-reference/cli-ref-restore.md) command, which downloads and installs any packages missing from the *packages* folder.
+The NuGet CLI [restore](../../reference/cli-reference/cli-ref-restore.md) command downloads and installs any packages that are missing from the packages directory.
 
-For projects migrated to PackageReference, use [msbuild -t:restore](../package-restore.md#restore-using-msbuild) to restore packages instead.
+Like `install`, the `restore` command only adds packages to disk, but doesn't change the project's dependencies. To change project dependencies, modify the *packages.config* file manually, then use `restore`.
```

Comment:
I think we can clarify that nuget.exe does support PackageReference as well. We just recommend using msbuild or dotnet restore baed on the project flavor. 

Now it kind of sounds like it doesn't.

Reply:
@nkolev92 - Added "you can." Do you think that's enough?

Reply:
Can you point where that was added? 
My idea was more about adding it in the description that mentions a packages.config rather than the note itself.

Reply:
@nkolev92 - I just changed the note to say "you can," meaning they don't have to, they can still use nuget restore if they want. 
![image](https://user-images.githubusercontent.com/20667102/186989245-e7d5c012-8c27-4feb-8300-4a63c4bfb83e.png)

However it sounds like you want larger revisions, maybe at the beginning of this article you want to not advise using PackageReference or dotnet CLI? 
![image](https://user-images.githubusercontent.com/20667102/186989039-a30a95ba-3950-40c3-a4a0-8cc838c1873f.png)

Please advise! Thanks.

Reply:
I'll try to cover what we our guidance is wrt to using the tools. 

NuGet.exe - supports packages.config and PackageReference, both SDK and non-SDK PackageReference. 
MSBuild.exe - Supports PackageReference, both SDK and non-SDK PackageReference and packages.config next to project through an opt-in
dotnet.exe - Supports PackageReference, only SDK PackageReference. Does not support packages.config. 

When people have PackageReference, we prefer that they use msbuild.exe or dotnet.exe, respectively. 

For example, SDK projects -> dotnet.exe
non-SDK PackageReference - msbuild.exe
mix of SDK and non-SDK -> msbuild.exe. 

We don't have to clarify all of that. 

My thought was:

- say nuget.exe can restore both packages.config and PackageReference
- say that if they have non-SDK PackageReference projects, they should msbuild.exe -t restore.

Reply:
Clarified this in both the Restore and Install sections.